src/terrain/globe_raster_program.js

```diff
@@ -78,15 +92,29 @@ const globeRasterUniformValues = (
     globeMercatorMatrix: Float32Array,
     zoomTransition: number,
     mercCenter: [number, number],
-    gridMatrix: ?Array<number>
+    gridMatrix: ?Array<number>,
+    frustumDirTl: ?[number, number, number],
+    frustumDirTr: ?[number, number, number],
+    frustumDirBr: ?[number, number, number],
+    frustumDirBl: ?[number, number, number],
+    globePosition: ?[number, number, number],
+    globeRadius: ?number,
+    viewport: ?[number, number]
```

Comment:
Since these will be non-optional, following the suggestion to add this for the poles as well, let's move `gridMatrix` as the last parameter, so a call such as:
```
func(requiredParam1, requiredParam2, undefined);
```
can be simplified as:
```
func(requiredParam1, requiredParam2);
```